frontend/pages/model/[modelId]/template.tsx

```diff
@@ -0,0 +1,109 @@
+import { ArrowBack, FileCopy } from '@mui/icons-material'
+import { Autocomplete, Button, Card, Container, Stack, TextField, Typography } from '@mui/material'
+import { EntrySearchResult, useGetModel, useListModels } from 'actions/model'
+import { postFromTemplate } from 'actions/modelCard'
+import _ from 'lodash-es'
+import { useRouter } from 'next/router'
+import { SyntheticEvent, useState } from 'react'
+import Loading from 'src/common/Loading'
+import Title from 'src/common/Title'
+import ErrorWrapper from 'src/errors/ErrorWrapper'
+import Link from 'src/Link'
+import MessageAlert from 'src/MessageAlert'
+import { EntryKind, EntryKindLabel } from 'types/types'
+import { getErrorMessage } from 'utils/fetcher'
+
+export default function ModelTemplateSelect() {
+  const router = useRouter()
+  const { modelId }: { modelId?: string } = router.query
+
+  const [selectedModel, setSelectedModel] = useState<EntrySearchResult | null>(null)
+  const [submissionErrorText, setSubmissionErrorText] = useState('')
+
+  const {
+    model: entry,
```

Comment:
As this is currently only designed to work with models, consider using the "model" terminology instead of "entry" for all of these variables.

Reply:
Aren't we planning on changing the names in our actions to be entries? If so this will reduce future tech debt.